lib/owdit.js

```diff
@@ -152,40 +152,44 @@ function check(cwd, callback) {
       data:    flatList,
       headers: {"Content-Type": "application/json"}
     };
-    const request = client.post("https://ossindex.net/v2.0/package", restOptions, (data) => {
-      let numVulnerabilitiesFails = 0;   //eslint-disable-line no-magic-numbers
-      let numVulnerabilitiesWarns = 0;   //eslint-disable-line no-magic-numbers
-      const vulnerablePackages = data.filter((element) => {
-        if (warns.includes(element.name)) {
-          numVulnerabilitiesWarns += element["vulnerability-matches"];
-        } else {
-          numVulnerabilitiesFails += element["vulnerability-matches"];
-        }
-
-        return element["vulnerability-matches"] > 0;  //eslint-disable-line no-magic-numbers
-      }).map((element) => {
-        return {
-          name:            element.name,
-          warnOnly:        warns.includes(element.name) || warns.includes(`${element.name}@${element.version}`),
-          version:         element.version,
-          path:            registerModules[`${element.name}${element.version}`].path,
-          vulnerabilities: element.vulnerabilities.map((element) => {
-            return {
-              title:       element.title,
-              description: element.description,
-              versions:    element.versions,
-              references:  element.references
-            };
-          })
+    const request = client.post("https://ossindex.net/v2.0/package", restOptions, (data, response) => {
+      if(response.statusCode >= 500) {
```

Comment:
It is not clear why this is needed? Could you please add a comment in the code that explains why this is needed (and when it could happen for instance).